### DISCUSSION TITLE
Answer is right but still error

### DISCUSSION THREAD

**Discussion by ShubhSensei** [Cyfrin/foundry-full-course-cu#138] [Q&A]
hey, the answer for lesson 5 is "ipfs://QmarzFRs2c7prT7gWFF2BpcQ6tuZjy7TcV7AzsNTuxoVy8" because after doing keccak256 it gives me the expected bytes then why it gives me an error? where am I wrong?

<img width="897" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/91625601/2337fa52-ee45-4215-9aac-b31f6192d63d">

**Comment by palmcivet7** [comment]
Pay close attention to the comments in the contract:
```
* Use all the help you can on this one! Google, AI, friends, peeranha, ethereum stack exchange, etc. 
     * 
     * Hint: It's a very common...
     * 
     * @param password - A string that when you keccak256 it will return the `EXPECTED_BYTES`!
```
FWIW I found Google very helpful with this.

**Reply by Naman1729** [reply]
hey @SensationalShubham , You can take a look form previous **Transaction** .
By following these steps : -
1.  Click on **Transaction hash** .
2. Click on **"Click to show more details"** .
3. Click on **Decode input data** .
Hope you got your NFT 🥳.
